### ISSUE TITLE
Keyboard tooltip of toolStripDropDownbutton in toolStrip1 is displayed after expanded and collapsed the toolStripDropDownbutton in toolStrip2

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#12819]
### .NET version

10.0.100-alpha.1.25064.17

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

Repro in previous .NET versions

### Issue description

When expand and collapse the toolStripDropDownbutton in toolStrip2, it displays the keyboard tooltip of toolStripDropDownbutton in toolStrip1, not item's in toolStrip2:
Also repro between these controls: MenuStrip/ToolStrip/StatusStrip

![Image](https://github.com/user-attachments/assets/0160e462-4f1b-4658-a590-0da18f9982ad)

https://github.com/user-attachments/assets/3314d249-9d70-43e9-a561-f792a3e293d1

### Steps to reproduce

1.	Create Winforms .NET app with 2 toolStrips
2.	In these 2 toolStrips, add ToolStripDropDownButton, add toolStripMenuItem for ToolStripDropDownButton separately
3.	Set these 2 toolStrips' TabStop property to True
4.	Build and run app
5.	Press down to expand ToolStripDropDownButton in toolStrip1 - it will focus on toolStripMenuItem
6.	Press Alt key to collapse ToolStripDropDownButton in toolStrip1, tab to toolstrip2 button
7.	Press down to expand ToolStripDropDownButton in toolStrip2 - it will focus on toolStripMenuItem
8.	Press Alt key to collapse ToolStripDropDownButton in toolStrip2

[WinFormsApp81.zip](https://github.com/user-attachments/files/18488676/WinFormsApp81.zip)

**Comment by Tanya-Solyanik** [comment]
@Olina-Zhang - does this work on .NET Framework?

**Comment by Olina-Zhang** [comment]
> [@Olina-Zhang](https://github.com/Olina-Zhang) - does this work on .NET Framework?

It doesn't work in .NET framework, have same behavior.

**Comment by Philip-Wang01** [comment]
Verified the issue on latest .NET SDK 10.0.100-preview.2.25112.47 + private dlls built from the main branch of Winforms repo. the issue has been fixed that the Keyboard tooltip display well as below screenshot.

https://github.com/user-attachments/assets/7c7f7519-c825-41c3-b8e1-efc69315fece

**Comment by chaowendy** [comment]
Verified with .NET SDK 10.0 P2 test pass build, it was fixed as above testing result.
